y

```diff
@@ -0,0 +1,4 @@
+    Host git.slygods.com
+    HostName git.slygods.com
+    User git
+    IdentityFile ~/.ssh/id_rsa_slygods
```

Comment:
Why do you need this file?

Reply:
without this file the keys at work were constantly lost

Reply:
What do you mean lost? Basically work should be saved automatically without having any additional files, maybe you should make some google research to identify why your work is lost.

In previous task you added this file as well to save work?